app/src/main/java/org/sopt/at/component/topbar/AtSoptMainTopBar.kt

```diff
@@ -0,0 +1,79 @@
+package org.sopt.at.component.topbar
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.at.R
+import org.sopt.at.ui.theme.ATSOPTANDROIDTheme
+import org.sopt.at.ui.theme.TvingLogo
+
+@Composable
+fun AtSoptMainTopBar(
+    modifier: Modifier = Modifier,
+    shareTvIconClick: () -> Unit = {},
+    myIconClick: () -> Unit = {}
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(Color.Black)
+            .padding(20.dp),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.ic_tving_logo),
+            contentDescription = "TVING logo",
+            modifier = Modifier
+                .height(24.dp)
+        )
+
+        Row(){
+            Icon(
+                painter = painterResource(id = R.drawable.ic_maintopbar_share),
```

Comment:
Icon에서는 painterResource 대신 imageVector를 사용하는 게 권장됩니다!  

```suggestion
imageVector = vectorResource(id = ...)
```
로 수정하면 더 좋을 거 같아요!